rest-api/offline/sa_key_remove.py

```diff
@@ -31,6 +32,8 @@ def delete_service_account_keys():
 
           for key in keys:
             keyname = key['name']
+            if keyname in service_account_keys:
```

Comment:
So I may be wrong here, but I think what we're looking to compare is account['email'], not key['name']. You should replace the logic on line 25 above with this.

Reply:
correct. key['name'] is the "human readable" form i.e. `configurator` or `App Engine default`